packages/strapi-design-system/src/v2/SimpleMenu/Menu.tsx

```diff
@@ -29,17 +29,23 @@ interface TriggerProps extends ButtonProps {}
 
 const MenuTrigger = forwardRef<HTMLButtonElement, TriggerProps>(
   ({ size, endIcon = <CarretDown width={`${6 / 16}rem`} height={`${4 / 16}rem`} aria-hidden />, ...props }, ref) => {
+    const isSizeS = size === 'S';
+    const isIconButton = (props.as as any)?.displayName === 'IconButton';
```

Comment:
I'm not sure about this, I don't think you're really supposed to access those props.

Does it make sense for this component to adjust its styling based on the as prop which realistically could be any component.

Is this to fix an issue in cloud? How come you can't change the padding there? 🤔

Reply:
Yeah, I agree about the conditional with `displayName` is not very good.

But, look, I am trying to fix this problem here:
https://design-system-git-main-strapijs.vercel.app/?path=/story/design-system-components-v2-simplemenu--with-icons
A variation of `SimpleMenu`, which uses the `IconButton` as the menu trigger (instead of a regular text).
IDK when, but a recent update on the DS Button result in this "missing-icon" behavior here, where the padding don't let the SVG appear.

So, yes, I am trying to fix a visual issue for Strapi Cloud Dashboard too
😅